package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "sw3",
-  "version": "1.0.0",
+  "name": "sw3_with_ERC20",
```

Comment:
revert this

Reply:
done

Reply:
this is still different from the name in `package.json`